app/misc/repo.ts

```diff
@@ -344,7 +344,7 @@ function openRepository() {
   }
 
 // works as a monitor for any change to the reference list
-function refreshList(verbose) {
+function refreshReferences(verbose, force) {
```

Comment:
Solves #101. When refreshing the graph the branch and tag names get cleared in refreshAll() but if no changes are made to the references in git, then they will not be reloaded in refreshReferences(). The force flag will ensure the tags/branches are reloaded when the graph is refreshed.